.github/workflows/release-build.yml

```diff
@@ -222,10 +224,11 @@ jobs:
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
 
-          # Check if branch already exists
+          # Check if branch already exists and delete it
           if git ls-remote --exit-code --heads origin ${BRANCH_NAME} > /dev/null 2>&1; then
-            echo "Error: Release branch ${BRANCH_NAME} already exists. Please manually delete the branch first and then rerun the workflow."
-            exit 1
+            echo "Release branch ${BRANCH_NAME} already exists. Deleting it..."
+            git push origin --delete ${BRANCH_NAME}
+            echo "Deleted existing release branch: ${BRANCH_NAME}"
           fi
```

Comment:
Does this mean if a pypi package is already published by a previous release action, and the action got triggerred again, it will delete the previous release branch? Let's avoid automatic deletion here.